packages/fxa-auth-server/lib/senders/emails/bin/mjml-sever.ts

```diff
@@ -0,0 +1,56 @@
+import http from 'http';
```

Comment:
Kinda of strange to have this bin file here.  Could this move to the fxa-auth-server bin directory?